src/app/dataset/depublication/sortable-header/sortable-header.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { SortDirection, SortHeaderConf, SortParameter } from '../../../_models';
+
+@Component({
+  templateUrl: './sortable-header.component.html',
+  selector: 'app-sortable-header',
+  styleUrls: ['./sortable-header.component.scss']
+})
+export class SortableHeaderComponent {
+  isLocked = false;
+  statuses = Object.values(SortDirection);
+  classes = this.statuses.map((status: string) => status.toLowerCase());
```

Comment:
Would it be better to have, in each enum member, a constant indicating the css class value rather than compute it from the enum value? That way you're not tied by that value and can give the class a more meaningful name.

Reply:
I can see your point, but doesn't it clutter the code, and are the derived class names not going to be clear (by definition) on account of their one-to-one mapping with the status values?  If not would a prefix (or suffix) not bring clarity?

Reply:
Perhaps just adding a prefix would be the way to go indeed?

Reply:
Added the prefix `sort-`